examples/hello_world/android/settings.gradle.kts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file is currently NOT auto generated.
+// DO NOT update it by running by dev/tools/bin/generate_gradle_lockfiles.dart.
```

Comment:
this is a confusing comment. running that script won't update this anyway, right?

Reply:
or will it, and is the user supposed to run that script then revert the change to this file? If so, I would recommend we update the script instead.

Reply:
> or will it, and is the user supposed to run that script then revert the change to this file?

That's right.

I created #140115 because changing the `generate_gradle_lockfiles.dart` script and running it is a quite massive change (see that PR's "lines changed"). That's why this comment exists.

Reply:
done in 90c07d330b4a8a9dd6f67ca66f82117b2b64037c